autoload/SpaceVim/layers/lang/c.vim

```diff
@@ -132,13 +145,26 @@ function! SpaceVim#layers#lang#c#set_variable(var) abort
     let g:neomake_c_enabled_makers = ['clang']
     let g:neomake_cpp_enabled_makers = ['clang']
     let s:clang_executable = a:var.clang_executable
-    let g:asyncomplete_clang_executable = a:var.clang_executable
+    if !has('nvim')
```

Comment:
sometime there is no clang_executable in a:var

Reply:
> sometime there is no clang_executable in a:var

the above has already check the key, 
```
function! SpaceVim#layers#lang#c#set_variable(var) abort
  if has_key(a:var, 'clang_executable')
    let g:completor_clang_binary = a:var.clang_executable
    let g:deoplete#sources#clang#executable = a:var.clang_executable
    let g:neomake_c_enabled_makers = ['clang']
    let g:neomake_cpp_enabled_makers = ['clang']
    let s:clang_executable = a:var.clang_executable
    if !has('nvim')
      let g:asyncomplete_clang_executable = a:var.clang_executable
    endif
  endif
........
```
I just add  a conditional to this block
```
if !has('nvim')
   ...
endif
```